.dockerignore

```diff
@@ -0,0 +1,6 @@
+.git
+.dockerignore
+.idea
+bin
```

Comment:
We don't have `bin` and `docker` dirs here.

Reply:
We have bin dir for travis